src/assets/scripts/client/scope/RadarTargetCollection.js

```diff
@@ -154,7 +154,8 @@ export default class RadarTargetCollection extends BaseCollection {
         const radarTargetModels = this._items;
         const results = _filter(radarTargetModels, ({ aircraftModel }) =>
             aircraftModel.transponderCode === aircraftReference ||
-            aircraftModel.callsign === aircraftReference
+            aircraftModel.callsign === aircraftReference ||
+            aircraftModel.cid === aircraftReference
```

Comment:
Please remove the todo comment above `findRadarTargetModelForAircraftReference`

Reply:
good catch. done